Add documentation for Foreman KernelCare plugin (#1848)
theforeman/foreman-documentation: guides/common/modules/proc_live-patching-hosts-using-kernelcare.adoc

[id="Live_Patching_Hosts_Using_KernelCare_Plug_In_{context}"]
= Live Patching Hosts Using KernelCare Plug-In

.Prerequisites
. Ensure your host has the `kernelcare` package installed.

> . Ensure your hosts runs Debian 9, Debian 10, Debian 11, Ubuntu 18.,04, Ubuntu 20.04, Ubuntu 22.04, {EL} 7, {EL} 8, or {EL} 9.

Reviewer: ```suggestion
. Ensure your hosts runs Debian 9, Debian 10, Debian 11, Ubuntu 18.04, Ubuntu 20.04, Ubuntu 22.04, {EL} 7, {EL} 8, or {EL} 9.
```

[id="Live_Patching_Hosts_Using_KernelCare_Plug_In_{context}"]
= Live Patching Hosts Using KernelCare Plug-In

You can use KernelCare plug-in to live-patch the Linux kernel on managed hosts.

.Prerequisites

> * Ensure your host has the `kernelcare` package installed.
> * Ensure your hosts runs {EL} 7, {EL} 8, or {EL} 9.
> * Ensure your host has access to the internet to connect to cloudlinux.com.

Reviewer 2: I would stick either with singular or plural. Note that the title mentions multiple hosts.
```suggestion
* Ensure your hosts have the `kernelcare` package installed.
* Ensure your hosts run {EL} 7, {EL} 8, or {EL} 9.
* Ensure your hosts have access to the internet to connect to cloudlinux.com.
```

Author: Applied.
